kohese-portal/client/src/components/create-wizard/create-wizard.component.html

```diff
@@ -21,10 +21,10 @@
       <mat-card>
         <mat-card-content>
           <format-object-editor #formatObjectEditor
+            [allowKindChange]="true"
```

Comment:
This is weird.  Why would changing the order have an effect?  Maybe we need to put logic into the formatObjectEditor code to defer the input parameter that is clobbering the allowKindChange attribute instead...

Reply:
The issue has been resolved by deferring logic to set the selectedNamespace until allowKindChange has been set.